packages/amplify-graphql-transformer-core/package.json

```diff
@@ -52,6 +52,7 @@
         "@aws-cdk/custom-resources": "~1.119.0",
         "@aws-cdk/cx-api": "~1.119.0",
         "@aws-cdk/region-info": "~1.119.0",
+        "@types/md5": "^2.3.1",
```

Comment:
Also, it doesn't look like the package.json has the actual md5 module in it.

Reply:
updated